<?xml version="1.0"?>
<launch>
    <param name="use_sim_time" value="false" />
    <arg name="info_topic" value="/camera_info"/>
    <arg name="image_topic" value="/wide/left/image_raw"/>
    <arg name="queue_size_img" value="1"/>
    <arg name="queue_size_imu" value="10"/>

    <node pkg="undistort_images" type="publishCameraInfo" name="publishCameraInfo" output="screen">
        <param name="topic/info" value="$(arg info_topic)" />
        <param name="topic/img" value="$(arg image_topic)" />
        <param name="camfile_left" value="snapcam1" />
        <param name="queue_size/img" value="$(arg queue_size_img)" />
        <param name="queue_size/imu" value="$(arg queue_size_imu)" />
    </node>

</launch>
